### ISSUE TITLE
Scout users with more than one account might not see all their institutes

### ISSUE THREAD

**Issue by northwestwitch** [Clinical-Genomics/scout#1472]
A quick(ish) fix would be to manually copy all the missing institutes from the complete account to the one missing institiutes.

A less quick fix is to modify the adapter.user query to return the user which has most institutes.

The best way to fix this is to go thru all the users in the database, add all institutes to the official user account and remove the extra accounts.

**Comment by dnil** [comment]
The first is presumably the way to go, right?

Suggestion (2) seems like a bit of work for a rather unpredictable outcome anyway. What if the user instances have (partly) disjoint sets of institutes, but roughly the same number? Hard to justify code that works that way?! 

I would also feel somewhat uneasy about (3) without careful reading/testing of code around case assignments and event logging in general.

**Comment by northwestwitch** [comment]
There's another aspect of this multi-account mess: the associated events.
If you look at the users list here: https://scout.scilifelab.se/users you'll see that some double accounts were there but were never used (kind of). We should keep using the right account!

**Comment by northwestwitch** [comment]
I have the feeling that this could be solved by selecting preferably those users that have _id as an email address. This behavior could have been modified by #1457

**Comment by dnil** [comment]
> "Hi!
> 
> I tried to assign myself to a case, and while I see that happening in the activity log (for as many times as I clicked on "assign me"), my name does not show up in the main list of cases.
> 
> Kind regards,
> Bianca"

**Comment by dnil** [comment]
That answers worry under (3) above, and weighs towards that we would be better off without the dupe accounts.

**Comment by dnil** [comment]
Lets see, what do we have in the dbs in terms of ObjectId user_id connections?
**stage**
```
> db.user.find().count()
127
> db.user.find({'_id':{'$type':'objectId'}}).count()
39
db.event.find({'user_id':{'$type':'objectId'}}).count()
8
db.acmg.find({'user_id':{'$type':'objectId'}}).count()
0
db.clinvar.find({'user':{'$type':'objectId'}}).count()
0
db.clinvar_submission.find({'user_id':{'$type':'objectId'}}).count()
0
```
**prod**
```
> use scout
switched to db scout
> db.user.find().count()
139
> db.user.find({'_id':{'$type':'objectId'}}).count()
39
> db.event.find({'user_id':{'$type':'objectId'}}).count()
84
> db.acmg.find({'user_id':{'$type':'objectId'}}).count()
1
> db.clinvar.find({'user':{'$type':'objectId'}}).count()
0
> db.clinvar_submission.find({'user_id':{'$type':'objectId'}}).count()
0
```

**Comment by dnil** [comment]
So same users, and a bit more productive on the stage env, but not dramatic. We could edit that down manually, or script it, then fix the user documents.

**Comment by dnil** [comment]
Fixed by #1484
